src/Select/Select.tsx

```diff
@@ -53,13 +54,17 @@ const InnerSelect = React.forwardRef<HTMLDivElement, SelectInnerProps>(
       options,
       multi,
       value,
+      openOnHover,
     })
 
     const context = React.useMemo<SelectContextValue>(
       () => ({ multi, canSelectAll }),
       [multi, canSelectAll]
     )
 
+    const onMouseOver = actions.onHover
```

Comment:
Why don't you check the `openOnHover` here ?

Reply:
Because it is already checked inside the reducer.

Reply:
That's weird to check this behaviour in 2 places don't you think ?